sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Receipt.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+using Azure.AI.FormRecognizer.Models;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    /// <summary>
+    /// Represents a receipt recognized from the input document and provides strongly-typed members
+    /// for accessing common fields present in recognized receipts.
+    /// </summary>
+    public class Receipt
```

Comment:
it will be good to add the link to the document reference about what a receipt has so that user understands where the fields are coming from